src/library/scala/collection/IterableOnce.scala

```diff
@@ -643,9 +643,10 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
     *  $willNotTerminateInf
     */
   def copyToArray[B >: A](xs: Array[B], start: Int = 0): xs.type = {
```

Comment:
why do we need this method at all when we have an overload that could be used, with a change in the default parameters https://github.com/scala/scala/pull/6887/files#diff-615db7dda80047901a4140c44c526dc5R674


Note - there are inconsistent overrides of these methods currently. Should we not be simplifying the interface, rather than duplicating the behaviour for the cost of saving the math.min on line 677